cookbook/core/containerization/raw_container.py

```diff
@@ -4,71 +4,134 @@
 Using Raw Containers
 ---------------------
 
-This example shows how it is possible to use arbitrary containers and pass data between them using Flyte.
+This example demonstrates how to use arbitrary containers in 5 different languages, all orchestrated in flytekit seamlessly.
 Flyte mounts an input data volume where all the data needed by the container is available and an output data volume
 for the container to write all the data which will be stored away.
 
 The data is written as separate files, one per input variable. The format of the file is serialized strings.
-Refer to the raw protocol to understand how to leverage this
+Refer to the raw protocol to understand how to leverage this.
 """
+import logging
+from typing import Tuple, Any, Mapping, List, Set
+from flytekit import task, workflow
 from flytekit import ContainerTask, kwtypes, workflow
 
+logger = logging.getLogger(__file__)
+
 # %%
 # A :py:class:`flytekit.ContainerTask` denotes an arbitrary container. In the following example, the name of the task
-# is square. This name has to be unique in the entire project. Users can specify some
+# is ``calculate_ellipse_area_shell``. This name has to be unique in the entire project. Users can specify:
 #
 # - ``input_data_dir`` -> where inputs will be written to
 # - ``output_data_dir`` -> where Flyte will expect the outputs to exist.
 #
 # inputs and outputs specify the interface for the task, thus it should be an ordered dictionary of typed input and
-# output variables
-square = ContainerTask(
```

Comment:
for each task lets put a link to the source code. you can use `literalinclude` in a panel for this
```
.. literalinclude:
```

Reply:
@kumare3 , I added the contents of each script to a panel, like so: https://flyte--492.org.readthedocs.build/projects/cookbook/en/492/auto/core/containerization/raw_container.html. Was that what you had in mind?